app/src/main/java/com/zulip/android/filters/NarrowFilterByDate.java

```diff
@@ -56,23 +47,30 @@ public void writeToParcel(Parcel dest, int flags) {
     @Override
     public Where<Message, Object> modWhere(Where<Message, Object> where)
             throws SQLException {
-        where.like(Message.TIMESTAMP_FIELD, new SelectArg(date));
+        DateUtils.truncate(new Date(), java.util.Calendar.DAY_OF_MONTH);
```

Comment:
`DateUtils.truncate` returns an rounded date you are not using this why is this here?

https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/time/DateUtils.html#truncate(java.util.Calendar, int)